src/test/resources/application.properties

```
@@ -0,0 +1 @@
+policies-history.cleaner.period=disabled
```

This should be in the `src/main/application.properties` file with a `%test.` prefix:
```suggestion
%test.policies-history.cleaner.period=disabled
```

---

Ah, that's smarter. Thanks :)

---

Yeah both approaches work but having a single file is easier to maintain.